message EdgeType{
   required uint64 src = 1;
   required uint64 dst = 3;

   required double vRank = 4;
   required double rank = 5; // of src
   required uint64 nNbrs = 6; // of src
  }
message RecordType {
  required uint64 key = 1;
//  repeated uint64 values = 2;
  repeated EdgeType values = 2;

}

